lunatrace/bsl/hasura/migrations/lunatrace/1666796296711_set_fk_public_builds_release_id/up.sql

```diff
@@ -0,0 +1,5 @@
+alter table "public"."builds"
```

Comment:
maybe Im wrong but couldnt all of this and these two migrations just be 

```
ALTER TABLE "public"."builds" ADD COLUMN "release_id" uuid NULL REFERENCES package.release ON UPDATE restrict ON DELETE restrict;
```

Also if we stick to capital postgres that's probably for the win, that's what everything else is in the project more or less. At least everything the rest of us have wrote.

Reply:
Makes sense. These were generated from hasura but I'll format them using our standard and merge them.